tasks/node/remotetest.js

```diff
@@ -0,0 +1,340 @@
+'use strict';
+
+require('dotenv').config({
+	silent: true
+});
+
+const path = require('path');
+const wd = require('wd');
+const denodeify = require('denodeify');
+const readFile = denodeify(require('graceful-fs').readFile);
+const writeFile = denodeify(require('graceful-fs').writeFile);
+const testResultsPath = path.join(__dirname, '../../test/results');
+const testResultsFile = path.join(testResultsPath, 'results.json');
+const mkdirp = denodeify(require('mkdirp'));
+const argv = require('minimist')(process.argv.slice(2));
+const cli = require('cli-color');
+const pollTick = 1000;
+const testBrowserTimeout = 60000;
+const environment = ['browserstack', 'saucelabs'].filter(x => x in argv)[0] || 'browserstack';
+const testProvider = require(`./${environment}`);
+
+const browserSets = {
+	"quick": [
+		'chrome/54',
+		'firefox/49',
+		'ie/14',
+		'ie/11',
+		'ie/8',
+		'android/4.4'
+	],
+	"ci": [
+		'chrome/54',
+		'chrome/48',
+		'firefox/49',
+		'firefox/44',
+		'ie/14',
+		'ie/13',
+		'ie/11',
+		'ie/10',
+		'ie/9',
+		'ie/8',
+		'ie/7',
+		'safari/9',
+		'safari/8',
+		'android/4.4'
+	],
+	"full": [
+		'chrome/54',
+		'chrome/48',
+		'chrome/46',
+		'chrome/42',
+		'chrome/40',
+		'chrome/35',
+		'firefox/49',
+		'firefox/44',
+		'firefox/42',
+		'firefox/41',
+		'firefox/33',
+		'firefox/30',
+		'ie/14',
+		'ie/13',
+		'ie/11',
+		'ie/10',
+		'ie/9',
+		'ie/8',
+		'ie/7',
+		'safari/9',
+		'safari/8',
+		'safari/5.1',
+		'android/4.4',
```

Comment:
android 5.1 and ios/9.2 have been removed as browserstack have not added these devices to their automated testing suite yet. I spoke with them on Twitter and this was their response - https://twitter.com/browserstack/status/818835556968833024